iot/http_example/system-test/cloudiot_http_example_test.js

```diff
@@ -51,68 +51,58 @@ test.after.always(async () => {
 test(`should receive configuration message`, async (t) => {
   const localDevice = `test-rsa-device`;
   const localRegName = `${registryName}-rsa256`;
-  let output = await tools.runAsync(`${helper} setupIotTopic ${topicName}`, cwd);
-  output = await tools.runAsync(
+  await tools.runAsync(`${helper} setupIotTopic ${topicName}`, cwd);
+  await tools.runAsync(
       `${helper} createRegistry ${localRegName} ${topicName}`, cwd);
-  output = await tools.runAsync(
+  await tools.runAsync(
       `${helper} createRsa256Device ${localDevice} ${localRegName} resources/rsa_cert.pem`, cwd);
-  t.regex(output, new RegExp(`Created device`));
 
-  output = await tools.runAsync(
+  let output = await tools.runAsync(
```

Comment:
Nit: can these be `const` instead of `let`? (In general, use `const` for variables that are never reassigned to. If you make this change, please do so throughout your PR.)